src/cascadia/TerminalApp/TerminalSettingsSerializationHelpers.h

```diff
@@ -79,7 +79,7 @@ JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::TerminalControl::TextAntialiasing
 
 // Type Description:
 // - Helper for converting a user-specified closeOnExit value to its corresponding enum
-JSON_ENUM_MAPPER(::TerminalApp::CloseOnExitMode)
+JSON_ENUM_MAPPER(::winrt::TerminalApp::CloseOnExitMode)

```

Comment:
we don't _really_ need the `::` _before_ `winrt` right? There's not like a technical reason for that, right? We do that in front of some of the other namespaces in the global namespace because they conflict with other ones we've included (the `Microsoft` one comes to mind), but there's not like another `winrt` namespace we're conflicting with, right?

Reply:
No need for it. Every other reference to that namespace in this file did it though. But just did a good ol' Copy/Replace to fix that.

Reply:
I used explicit root namespace  anchoring because this is a _header_ (!) that might be included _anywhere_